nameparser/config/titles.py

```diff
@@ -7,12 +7,10 @@
     'brother',
     'dame',
     'father',
-    'king',
```

Comment:
Both king and queen are including in the set of titles that indicate first names when placed before a single name, e.g. King David and Queen Mary, so this pull request will break some tests. In 2005 there were 148 people born in the US named King, so maybe it is a more useful case to handle than the title. I'm know people have used this parser on datasets that include kings and queens before though, but I guess we can let them customize the titles constant to pick them up. 

We should update the test cases that include "king" to use one of the other titles in that set.